docs/jitserver_tuning.md

```diff
@@ -42,11 +42,14 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  To enable this feature, specify the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) command line option, both at the server and at the client JVM.
 
- A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). The default is to use a nameless cache.
+ A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache called `default`.
+
+ The maximum amount of memory that an AOT cache can consume at the server is 300 MB. Since each cache consumes memory, the number of caches that a JITServer can hold is limited by the memory available.
+
+ You can limit the amount of memory used by the JITServer AOT cache with the [`-XX:JITServerAOTmx=<size>`](xxjitserveraotmx.md) option. Once the cache size limit is reached, new clients will not be able to create new AOT cache instances or add new compiled methods to the existing AOT cache instances.
 
  Current limitations:
 
- - The amount of memory that an AOT cache can consume at the server is not limited. The number of caches that a JITServer can hold is also not limited.
  - The AOT cache is a non-persistent in-memory cache. If the JITServer instance ends, the cache content is lost.
```

Comment:
We will have another item that removes this first limitation.

Reply:
Since this issue is related to limiting the amount of memory used by the JITServer AOT, this limitation was removed. Hope that is ok.